alf/utils/common.py

```diff
@@ -147,17 +148,23 @@ def add_action_summaries(actions, action_specs):
     """
     action_specs = tf.nest.flatten(action_specs)
     actions = tf.nest.flatten(actions)
+
     for i, (action, action_spec) in enumerate(zip(actions, action_specs)):
         if len(action_spec.shape) > 1:
             continue
-        if len(action_spec.shape) == 0:
-            action_dim = 1
+        if tensor_spec.is_discrete(action_spec):
```

Comment:
line 155-158 should be in the loop of 165 in case it's multi-dimensional.

Reply:
action_dim is 1 for discrete actions